build.gradle.kts

```diff
@@ -0,0 +1,62 @@
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat
+import org.gradle.api.tasks.testing.logging.TestLogEvent
+
+plugins {
+    id("java-gradle-plugin")
+    id("com.gradle.plugin-publish") version "0.12.0"
+    groovy
```

Comment:
**Could**: Add a `// TODO remove` comment to show that the target of this is to remove groovy?